maven-central-publish/src/main/java/tech/yanand/gradle/PublishToCentralPortalTask.java

```diff
@@ -100,6 +105,12 @@ void executeTask() throws InterruptedException, IOException {
         }
     }
 
+    private boolean isSuccessful(String deploymentStatus) {
```

Comment:
Sorry, I forgot to remind you to put private after the public method.

Reply:
Ah yes, I have moved it near the other private methods.